### PR TITLE
Ruby

### PR DESCRIPTION
**Pull request by simanthastew**

### CODE REVIEW

**Comment by simanthastew** [comment]
@mechamanny - i would love a code review! i had particular trouble with release 4, the allergy loop. it seemed weird to me to set allergies = gets.chomp twice, but if i didn't set it outside of the loop i couldn't call on it later on. and if i couldn't include it in my last conditional block, it would say "Probably a vampire" and then the result of the conditional logic from the rest of the code. thanks in advance!!!

File: ruby/vampires.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    else
+      name = true
+    end
+
+  puts "In years, how old are you?"
+  age = gets.chomp.to_i
+  puts "What year were you born?"
+  year = gets.chomp.to_i
+    if age == (2016 - year)
+        then age = TRUE
```
Comment (mechamanny) [inline ruby/vampires.rb]: Keep `true` lowercase for consistent style.

File: ruby/vampires.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  puts "Would you like to enroll in our company's health insurance? Y/N"
+  insurance = gets.chomp
+  if insurance == "Y"
+    insurance = TRUE
+  else
+    insurance = FALSE
+  end
+
+  puts "If you have allergies, please list them, hitting enter after each one. When finished, type 'done'."
+  allergies = gets.chomp
```
Comment (mechamanny) [inline ruby/vampires.rb]: It is fine to call this more then once since the logic changes from the first time you gather the input to the other iterations.

**Comment by mechamanny** [comment]
Overall a solid submission!

**Pull request merged**